### ISSUE TITLE
ERROR: LoadError: UndefVarError: @formula not defined

### ISSUE THREAD

**Issue by lauswullum** [JuliaStats/Survival.jl#38]
Hi, Thanks for writing this package.
### Potential bug
I found a potential bug, where `@formula` from `StatsModels.jl` is not automatically available 
when only writing 
`using Survival` 
without 
`using StatsModels`

### MWE to reproduce the problem
```julia
using Survival
using StatsBase
using DataFrames
event = EventTime.(10 .+ randn(10), sample([true, false], 10))
coxph(@formula(events ~ x), DataFrame(x = randn(10), events = event))
```
output:
```julia
ERROR: LoadError: UndefVarError: @formula not defined
```

**Comment by ararslan** [comment]
Good catch. Not a bug per se but it would be a good improvement to re-export `@formula` from StatsModels. Should be as simple as adding `@formula` to the `export` list. This would also allow us to remove the dependency on StatsModels in the `docs/` project.
